docs/guides/security/audit_reports.mdx

```diff
@@ -103,4 +103,31 @@ Audits for the Babylon Genesis Chain are conducted by [Coinspect](https://www.co
   </BrowserOnly>
 </CardSection>
 
+### Babylon Genesis v2 Audits
 
+Audits for the [Babylon Genesis v2 Upgrade](https://babylonlabs.io/blog/babylon-genesis-v2-upgrade) are conducted by [Oak Security GmbH](https://oaksecurity.io/) and [Informal Systems](https://informal.systems/).
+
+<CardSection id="Babylon Security Audit Reports">
+  <BrowserOnly>
+    {() => (
+      <Card
+        title="Oak Security GmbH Audit Report"
+        to={new URL('/assets/files/oak_security_gmbh_genesis_v2_audit-2025_06.pdf', window.location.origin).href}
+        description="Oak Security GmbH Babylon Genesis v2 Upgrade Audit Report (June 2025)"
+        target="_blank"
+        rel="noopener noreferrer"
+      />
+    )}
+  </BrowserOnly>
+  <BrowserOnly>
+    {() => (
+      <Card
+        title="Genesis v2 Upgrade Audit Report"
+        to={new URL('/assets/files/informal_system_genesis_v2_audit_2025-06.pdf', window.location.origin).href}
```

Comment:
the filename had a hyphen and not underscore at the end. 

@kkkk666 Can we fix?